lib/WUI/wui.c

```diff
@@ -97,9 +101,8 @@ void StartWebServerTask(void const *argument) {
             switch (evt.value.v) {
             case EVT_TCPIP_INIT_FINISHED:
                 netdev_set_up(NETDEV_ETH_ID);
-                break;
-            case EVT_LWESP_INIT_FINISHED:
                 netdev_set_up(NETDEV_ESP_ID);
+                netdev_join_ap(NETDEV_ESP_ID);
```

Comment:
Should we join AP first before trying to set it up?

Reply:
Calls switched